app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -21,14 +28,42 @@ import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp52
 import co.nimblehq.compose.crypto.ui.theme.Style
 import co.nimblehq.compose.crypto.ui.theme.Style.textColor
 import co.nimblehq.compose.crypto.ui.uimodel.CoinItemUiModel
+import co.nimblehq.compose.crypto.ui.userReadableMessage
 
 @Suppress("FunctionNaming", "LongMethod")
 @Composable
 fun HomeScreen(
     viewModel: HomeViewModel = viewModel()
 ) {
+    val context = LocalContext.current
+    LaunchedEffect(Unit) {
+        viewModel.error.collect { error ->
+            val message = error.userReadableMessage(context)
+            Toast.makeText(context, message, Toast.LENGTH_SHORT).show()
+        }
+    }
+
+    val showMyCoinsLoading: IsLoading by viewModel.showMyCoinsLoading.collectAsState()
+    val showTrendingCoinsLoading: IsLoading by viewModel.showTrendingCoinsLoading.collectAsState()
     val myCoins: List<CoinItemUiModel> by viewModel.myCoins.collectAsState()
+    val trendingCoins: List<CoinItemUiModel> by viewModel.trendingCoins.collectAsState()
```

Comment:
Seems we need to `remember` this since we will have to handle the loading more.

Reply:
I think we can handle it later when we have to work on `loading more` feature 🤔

Reply:
I think so @Wadeewee @hoangnguyen92dn 😄